src/allotropy/parsers/utils/values.py

```diff
@@ -16,7 +16,7 @@
 PrimitiveValue = str | int | float
 
 
-def str_to_bool(value: Any) -> bool:
+def str_to_bool(value: str) -> bool:
     return str(value).lower() in ("yes", "y", "true", "t", "1")
```

Comment:
nit: don't need to cast `value` anymore in that case?

Reply:
Very true